src/cs50.c

```diff
@@ -57,7 +57,6 @@
 #undef get_int
 #undef get_long_long
 #undef get_string
-#define null (&(struct prompt) {0})
```

Comment:
Commented on the Asana task. With a couple of modifications to `cs50.c` we could swap the order such that the file looks like:

```
cs50.c
cs50.h
STUDENT CODE
```

Alternatively we could define another header file which just contains these `defines` and include it in both `cs50.h` and at the end of `cs50.c`